predicthq/kafka/consumer.py

```diff
@@ -49,26 +46,26 @@ def _group_messages(messages):
     return batch_ref
 
 
-class PhqKafkaConsumer(object):
+class Consumer(object):
     def __init__(
-        self, input_topic: str, group_id: str, batch_size: int, consumer_timeout_ms: int,
-        kafka_bootstrap_servers: List[str], kafka_consumer_config: Dict[str, str] = None
+        self, kafka_bootstrap_servers: List[str], input_topic: str, group_id: str,
```

Comment:
just one thing, why change kafka_bootstrap_servers from List to Str? I think it's better to provide a List and then do the 
 'bootstrap.servers': ",".join(bootstrap_servers)

Reply:
(never mind, it's early and I miss read it :P )